test/MemoryReference.hs

```diff
@@ -177,6 +178,12 @@ prop_parallel bug = forAllParallelCommands sm' $ \cmds -> monadicIO $ do
     where
       sm' = sm bug
 
+prop_general_parallel :: Bug -> Int -> Property
```

Comment:
```suggestion
prop_generalParallel :: Bug -> Int -> Property
```

The idiom to use the `prop_` snake case prefix comes from QuickCheck's `quickCheckAll` template Haskell code that scans the file for properties with that prefix, we should use Haskell's camel case for the rest of the name.